src/hotspot/share/oops/constantPool.hpp

```diff
@@ -342,9 +319,12 @@ class ConstantPool : public Metadata {
     *int_at_addr(which) = ((jint) name_and_type_index<<16) | bsms_attribute_index;
   }
 
+  // Note that release_tag_at_put is not needed here because this is called only when
+  // constructing a ConstantPool in a single thread, with no possibility of concurrent access.
```

Comment:
Can we put the comment inside the method please, prior to the `tag_at_put` call.